package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "webapis",
+  "private": "true",
+  "namespace": "@arcstack",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ArcStackLab/webapis.git"
+  },
+  "packageManager": "pnpm@9.2.0",
+  "description": "A Comprehensive Integration of All Web Standard APIs",
+  "scripts": {
+    "lint": "prettier . --check && eslint",
+    "format": "prettier . --write",
+    "uninstall-husky": "pnpm uninstall husky --no-save && git config --unset core.hooksPath && pnpm dlx rimraf .husky"
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "pnpm dlx lerna version && git add .",
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*": [
+      "eslint --fix",
+      "prettier --write"
```

Comment:
Running both eslint and prettier on all files might be redundant. This can be optimized by configuring eslint to handle prettier formatting.

```diff
-      "eslint --fix",
-      "prettier --write"
+      "eslint --fix --ext .js,.jsx,.ts,.tsx"
```